spring-grpc-spring-boot-autoconfigure/src/main/java/org/springframework/grpc/autoconfigure/server/GrpcServerProperties.java

```diff
@@ -35,13 +37,13 @@ public class GrpcServerProperties {
 	/**
 	 * Server address to bind to. The default is any IP address ('*').
 	 */
-	private String address = ANY_IP_ADDRESS;
+	private String host = ANY_IP_ADDRESS;
 
 	/**
 	 * Server port to listen on. When the value is 0, a random available port is selected.
 	 * The default is 9090.
 	 */
-	private int port = 9090;
+	private int port = GrpcUtils.DEFAULT_PORT;
```

Comment:
The generated config props is pretty smart, but it does not traverse class constants and leaves [default empty](https://github.com/spring-projects-experimental/spring-grpc/pull/30/files#diff-243856d43cde6a15728d5897a413d1f7c99b2743e8db88d938ea50889cd6ba0eR30). 

<sub>From [here](https://docs.spring.io/spring-boot/specification/configuration-metadata/annotation-processor.html)</sub>
> <sub>The annotation processor applies a number of heuristics to extract the default value from the source model. Default values have to be provided statically. In particular, do not refer to a constant defined in another class. Also, the annotation processor cannot auto-detect default values for Enums and Collections.</sub>

In order to get this default value into the metadata, we just need to add this to the [add. metadata](https://docs.spring.io/spring-boot/specification/configuration-metadata/annotation-processor.html#appendix.configuration-metadata.annotation-processor.adding-additional-metadata). Not sure if we have already created an add. metadata file. If you don't want to mess w/ this I am more than happy to handle this in a polish and merge. Just lmk

Reply:
I don't know how to do that yet. We'll do it later.

Reply:
I'll drop one in - no worries.